portfolio/src/main/java/com/google/sps/servlets/CommentsController.java

```diff
@@ -249,4 +260,19 @@ private String parseOrderByParam(String orderBy) {
       return "createdAt";
     }
   }
+
+  private Sentiment analyzeSentiment(String comment) throws IOException {
+    try (LanguageServiceClient languageService = LanguageServiceClient.create()) {
+      Document document = Document.newBuilder().setContent(comment).setType(Document.Type.PLAIN_TEXT).build();
+
+      double score = (double) languageService.analyzeSentiment(document).getDocumentSentiment().getScore();
+
+      if (score >= -1 && score <= -0.3)
```

Comment:
What do you think about making some constants for these so that it's clear what the thresholds are / for readability? And maybe some comment on why these numbers?

Reply:
Sounds great, actually I added some other variables to a `common` folder intended to save this kind of hardcoded data.

I'm going to merge this PR, but you can check it out to see if its okay